### DISCUSSION TITLE
Current Charge View -> Vis

### DISCUSSION THREAD

**Discussion by swiffer** [jheredianet/Teslamate-CustomGrafanaDashboards#97] [Ideas]
Hi @jheredianet - had a PV surcharge session today - would prefer setting the "Line Interpolation" setting of the graph to "Step After" - what do you think?

before:
![grafik](https://github.com/user-attachments/assets/c0fd1a0e-ac8c-434b-bf8b-17e9dac9698f)

after:
![grafik](https://github.com/user-attachments/assets/c2a51f91-7871-4481-9b3b-345f103c1c02)

**Comment by jheredianet** [comment]
Sure, it seems to be a better way to show the graph 👍🏼
